packages/coinstac-ui/app/render/components/dashboard-home.js

```diff
@@ -50,6 +51,16 @@ class DashboardHome extends Component {
       .then((responses) => {
         const remoteResults =
           sortBy(flatten(responses), ['endDate', 'startDate']).reverse();
+
+        /**
+         * Combat some weird React internal bug where `componentWillUnmount`
+         * never fires.
+         */
+        if (this._calledComponentWillUnmount) {
+          clearInterval(this.interval);
```

Comment:
There was some race condition and code in `componentWillUnmount` was never fired, occasionally causing `this.setState` errors on this component after it was unmounted.